go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/hashicorp/boundary-plugin-minio
 
-go 1.22.2
+go 1.23.4
+
+replace github.com/hashicorp/boundary-plugin-minio/madmin => ./madmin
```

Comment:
This replace seems like a noop? Isn't this already the module path of this module?

Reply:
I need to merge this then cut a tag of the madmin package to pull it in still, unless I am missing something

Reply:
Ah, that's fair